class-tgm-plugin-activation.php

```diff
@@ -2093,6 +2095,19 @@ public static function get_instance() {
 
 			return self::$instance;
 		}
+
+		/**
+		 * Adds CSS to admin head.
+		 *
+		 * @since 2.6.2
+		 */
+		function admin_css() {
```

Comment:
The CSS should only be added when the TGMPA page is being loaded, not for any other admin pages.

```php
			if ( ! $this->is_tgmpa_page() ) {
				return;
			}
```